src/cc2olx/models.py

```diff
@@ -451,58 +461,54 @@ def parse_lifecycle(self, node):
         data['contribute_date'] = text
         return data
 
-    def parse_organizations(self, node):
+    def _parse_organizations(self, node):
         data = []
-        element = node.find('ims:organizations', self.ns) or []
-        data = [
-            self.parse_organization(org_node)
-            for org_node in element
-        ]
+        element = node.find("ims:organizations", self.ns) or []
+        data = [self._parse_organization(org_node) for org_node in element]
```

Comment:
Nit: I don't expect you to revert it at this point, but in the future, it's generally helpful to avoid superficial syntax changes, unless there's a "good reason" (I know, I know, there's a grey area). They're easy places to introduce/overlook regressions, for no user gain and subjective developer gain (there's a good chance I wrote the list comprehension like that because I find them more legible and easier to extend :grin: ).

Reply:
>I don't expect you to revert it at this point

Perfectly fine! I just missed this while reverting other formatting changes.